mifosng-android/src/main/java/com/mifos/mifosxdroid/AboutActivity.java

```diff
@@ -0,0 +1,44 @@
+package com.mifos.mifosxdroid;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.View;
+import androidx.annotation.Nullable;
+import com.mifos.mifosxdroid.core.MifosBaseActivity;
+
+public class AboutActivity extends MifosBaseActivity {
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_about);
+        showBackButton();
+    }
+
+    public void goToweb (View view) {
+        String url = "https://github.com/openMF/android-client/graphs/contributors";
+        link(url);
```

Comment:
It would be better to use "@OnClick" with butterknife instead (look for other existing fragments or activities for reference such as ClientDetailsFragment). 
Change this for every other function.

Reply:
Is it necessary to in this way?

Reply:
It's the best way to do it, the method you used isn't frequently (if not never) used in the app's code. (also if you find any difficulty implementing that feature feel free to ask !)